zkevm-circuits/src/evm_circuit/util/constraint_builder.rs

```diff
@@ -1378,13 +1360,7 @@ impl<'a, F: Field> ConstraintBuilder<'a, F> {
         ret
     }
 
-    pub(crate) fn add_constraints(&mut self, constraints: Vec<(&'static str, Expression<F>)>) {
-        for (name, constraint) in constraints {
-            self.add_constraint(name, constraint);
-        }
-    }
-
-    pub(crate) fn add_constraint(&mut self, name: &'static str, constraint: Expression<F>) {
+    /*pub(crate) fn add_constraint(&mut self, name: &'static str, constraint: Expression<F>) {
```

Comment:
Shouldn't this be deleted instead of commented?

Reply:
Yes, that I solved in the next commit 👍